src/garage/torch/algos/ddpg.py

```diff
@@ -116,8 +108,8 @@ def __init__(
         self._buffer_batch_size = buffer_batch_size
         self._discount = discount
         self._reward_scale = reward_scale
-        self.max_episode_length = max_episode_length
-        self._max_episode_length_eval = max_episode_length_eval
+        self.max_episode_length = env_spec.max_episode_length
```

Comment:
```suggestion
        self._max_episode_length = env_spec.max_episode_length
```

Reply:
This should be public because it is accessed by `LocalRunner`.